src/PIL/ImageOps.py

```diff
@@ -601,10 +601,12 @@ def exif_transpose(image):
                     "Raw profile type exif"
                 ] = transposed_exif.tobytes().hex()
             elif "XML:com.adobe.xmp" in transposed_image.info:
-                transposed_image.info["XML:com.adobe.xmp"] = re.sub(
+                for pattern in (
                     r'tiff:Orientation="([0-9])"',
-                    "",
-                    transposed_image.info["XML:com.adobe.xmp"],
-                )
+                    r"<tiff:Orientation>([0-9])</tiff:Orientation>",
```

Comment:
I've noticed that the tests pass without this change. This is a side effect of #6335. I've created https://github.com/bigcat88/Pillow/pull/2 to fix the problem.